projects/sandbox/train/train.py

```diff
@@ -37,7 +40,7 @@ def load_signals(waveform_dataset: Path, parameter_names: List[str]):
                     values = np.log10(values)
                 data.append(values)
 
-        intrinsic = np.column_stack(data)
+        intrinsic = np.row_stack(data)
```

Comment:
Was this a bug previously?

Reply:
Not a bug: This is a result of moving to `ChannelWiseScaler` in ml4gw (see comment in `preprocessor.py`)